CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Skipper
+
+## Pull requests only
+
+**DON'T** push to the master branch directly. Always use feature branches and let people discuss changes in pull requests.
+Pull requests should only be merged after all discussions have been concluded and at least 1 reviewer has given their 
+**approval**.
+
+## Guidelines
+
+- **every change** needs a test
+- 100% code coverage
```

Comment:
100% code coverage is a bit unreasonable most of the times, I would remove this guideline.